src/app/api/core/services/policies.service.ts

```diff
@@ -15,6 +15,7 @@ export class PoliciesService extends BaseService {
       [Resource.WorkflowState]: [UserAction.Read],
       [Resource.ViewSetting]: [],
       [Resource.Users]: [],
+      [Resource.Attachments]: [],
```

Comment:
Clients are authorized to read attachments. Please add `UserAction.Read` else the client view of tasks app will be completely broken :bow: